tests/functional/ubuntu/16.04/mon/initial_members/vagrant_variables.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+# DEPLOY CONTAINERIZED DAEMONS
+docker: false
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
+osd_vms: 0
+mds_vms: 0
+rgw_vms: 0
+nfs_vms: 0
+rbd_mirror_vms: 0
+client_vms: 0
+iscsi_gw_vms: 0
+
+# Deploy RESTAPI on each of the Monitors
+restapi: true
```

Comment:
Oh wait this won't be used since we don't use the Vagrantfile for provisioning.
Since we don't can't we just keep this file simple and only use the number of VMs we want?